fast_carpenter/masked_tree.py

```diff
@@ -118,3 +127,137 @@ def mask_df(df, mask, start_event):
     else:
         masked = df.loc[mask]
     return masked
+
+
+class MaskedTrees(object):
+    nonOverloadedFunctions = [
+        'pandas', 'array', 'arrays', 'mask', 'unmasked_array',
+        'unmasked_arrays', 'apply_mask', 'trees', '_mask', 'event_ranger',
+        '_index', '__getitem__', 'reset_mask', 'new_variable', 'keys',
+    ]
+
+    def __init__(self, trees, event_ranger, mask=None):
+        self.pandas = types.SimpleNamespace(df=self._df)
```

Comment:
If this works just as well, then it's definitely neater than the implementation for the single tree case above.  Is it worth rewriting that so it's the same approach (for consistency and since this looks simpler)?

Reply:
Fixed in ee9bee5.